File: Python/1-if-else/exercicio15.py
"""
15 - Usando switch, escreva um programa que leia um inteiro entre 1 e 7 e imprima o dia da semana correspondente a este
número. Isto é, domingo se 1, segunda-feira se 2, e assim por adiante.
"""

print('Digite um número: ')
numero = int(input())

if 1 <= numero <= 7:
    if numero == 1:
        print('Domingo')
    if numero == 2:
        print('Segunda')
    if numero == 3:
        print('Terça')
    if numero == 4:
        print('Quarta')
    if numero == 5:
        print('Quinta')
    if numero == 6:
        print('Sexta')
    if numero == 7:
        print('Sábado')
else:
    print('Número inválido')

File: Java/1-if-else/src/if_else/Exercicio15.java
package if_else;

/*
15 - Usando switch, escreva um programa que leia um inteiro entre 1 e 7 e imprima o dia da semana correspondente a este
número. Isto é, domingo se 1, segunda-feira se 2, e assim por adiante.
*/

import java.util.Scanner;

public class Exercicio15 {
    public static void main(String[] args) {

        Scanner scan = new Scanner(System.in);

        System.out.println("Digite um numero de 1 a 7: ");
        int dia = scan.nextInt();

        switch (dia){
            case 1:
                System.out.println("Domingo");
                break;
            case 2:
                System.out.println("Segunda");
                break;
            case 3:
                System.out.println("Terça");
                break;
            case 4:
                System.out.println("Quarta");
                break;
            case 5:
                System.out.println("Quinta");
                break;
            case 6:
                System.out.println("Sexta");
                break;
            case 7:
                System.out.println("Sábado");
                break;
            default:
                System.out.println("Dia não existe");
        }
    }
}
